libbeat/dashboards/import_dashboards.go

```diff
@@ -211,7 +211,7 @@ func (imp Importer) CreateIndex() error {
 			},
 		})
 	if err != nil {
-		fmt.Printf("fail to set the mapping. Error: %s\n", err)
+		fmt.Printf("failed to set the mapping. Error: %s\n", err)
```

Comment:
I wonder why this is only logging an error and not returning it. @monicasarbu do you know?

We should write this to stderr if it's not going to be returned.